### PR TITLE
Support ECS Atlantis

### PR DESCRIPTION
**Pull request by osterman**
## what
* Support ECS Atlantis

## why
* CI/CD of Terraform

### CODE REVIEW

**Comment by osterman** [comment]
atlantis plan -d conf/tfstate-backend

**Comment by osterman** [comment]
atlantis plan -d conf/tfstate-backend

**Comment by osterman** [comment]
atlantis plan -d conf/tfstate-backend

**Comment by osterman** [comment]
atlantis plan -d conf/tfstate-backend

**Comment by osterman** [comment]
atlantis plan -d conf/tfstate-backend

**Comment by osterman** [comment]
atlantis plan -d conf/tfstate-backend

**Comment by osterman** [comment]
atlantis apply -p tfstate-backend

**Comment by osterman** [comment]
atlantis plan -d conf/tfstate-backend

File: Dockerfile [hunk 1]
```diff
@@ -2,7 +2,8 @@ FROM cloudposse/terraform-root-modules:0.5.7 as terraform-root-modules
 
 FROM cloudposse/helmfiles:0.7.2 as helmfiles
 
-FROM cloudposse/geodesic:0.32.8
+FROM cloudposse/geodesic:0.59.0
+#FROM cloudposse/geodesic:dev
```
Comment (joshmyers) [inline Dockerfile:7]: Commented code should not be committed.

File: atlantis.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  apply_requirements:
+    - "approved"
+
+workflows:
+  make:
+    plan:
+      steps:
+      - run: "export | grep TF_"
+      #- run: "direnv exec . init-terraform"
+      #- run: "direnv exec . make plan"
```
Comment (joshmyers) [inline atlantis.yaml]: Commented code should not be committed. I do not know the intent of this change. _why_ has this been commented out?

Comment (osterman) [reply atlantis.yaml]: agree - was just debugging/triaging. getting back to this now.

**Comment by osterman** [comment]
atlantis plan -d conf/tfstate-backend

**Comment by osterman** [comment]
atlantis plan -d conf/tfstate-backend

**Comment by osterman** [comment]
atlantis plan -d conf/tfstate-backend

**Comment by osterman** [comment]
Closing this as a "dirty" PR =)

**Pull request closed without merging**